src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,20 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+interface TodoInfoProps {
+  todo: Todo;
+}
+
+export const TodoInfo: React.FC<TodoInfoProps> = ({ todo }) => (
+  <article
+    className={classNames('TodoInfo', {
+      'TodoInfo--completed': todo.completed,
+    })}
```

Comment:
Also, you can make it libs variable the shortest
just use `import cn from classnames`
just for your information
```suggestion
    className={cn('TodoInfo', {
      'TodoInfo--completed': todo.completed,
    })}
```